src/decode.h

```diff
@@ -1161,6 +1164,18 @@ void DecodeUnregisterCounters(void);
 
 #define PKT_SET_SRC(p, src_val) ((p)->pkt_src = src_val)
 
+#define PKT_MAX_DECODED_LAYERS 16
```

Comment:
How did you come to 16?

Reply:
Looked reasonable to me, nothing scientific...
Do you have a better idea ?